performance/index.html

```diff
@@ -40,6 +40,14 @@
 <script src="jsonParsers/nearley/nearley_json_parser_tokenizer.js"></script>
 <script src="jsonParsers/nearley/api.js"></script>
 
+<script src="lib/jsonify/jsonify.js"></script>
```

Comment:
can we use:
https://unpkg.com/jsonify@0.0.0/lib/parse.js ?

and reduce 3rd party code committed to this repo?